### ISSUE TITLE
SIPS workshop help

### ISSUE THREAD

**Issue by jmansour99** [anamartinovici/githubfun#64]
Hi! I'm doing the SIPS workshop and would like to be added as a collaborator for the githubfun repo.

**Comment by anamartinovici** [comment]
Hi @jmansour99 

I have added you as a collaborator. You need to accept the invite and then you should be able to push to this repository.
If it all works fine, can you please close this issue? I have assigned you as 'responsible' for this, just to practice the feature.

**Comment by anamartinovici** [comment]
No reply - I am now closing this issue myself. If you have questions, you can reopen the issue or open a new issue.

**Comment by jmansour99** [comment]
Hi Ana,

sorry, I've not had any chance to look back at the sips content. I'm so sorry. When I do, probably not until this summer, I will certainly get in touch if I am still struggling.

Thanks again!
Jamal.

---------------------------------------------------------------------------------
Jamal K. Mansour, PhD
Senior Lecturer in Psychology

Psychology, Sociology, & Education
Queen Margaret University
Edinburgh, UK
EH21 6UU

Email: ***@***.***
Phone: +44 (0) 131 474 0000 and say my name (Jam-el Man-sir) when prompted
Fax: +44 (0) 131 474 0001

Web: https://www.qmu.ac.uk/schools-and-divisions/psychology-and-sociology/psychology-and-sociology-staff/dr-jamal-k-mansour/
Memory Research Group Web site: https://memoryresearchgroup.wordpress.com/
Twitter: @EyewitnessIDUp

Check out my new article on eyewitness identification confidence: https://www.sciencedirect.com/science/article/abs/pii/S2211368120300048
________________________________
